### ISSUE TITLE
Dimmer not adding

### ISSUE THREAD

**Issue by briandye** [damianxd/homebridge-vera#43]
I pulled down latest through NPM and am getting errors on my dimmer:

`Unhandled rejection TypeError: Object dimmer has no method 'includes'
    at /home/engineer/node_modules/homebridge-vera/lib/functions.js:159:55
    at Array.forEach (native)
    at Object.module.processdevices (/home/engineer/node_modules/homebridge-vera/lib/functions.js:153:14)
    at /home/engineer/node_modules/homebridge-vera/lib/functions.js:113:38
    at Array.forEach (native)
    at Object.module.processrooms (/home/engineer/node_modules/homebridge-vera/lib/functions.js:89:24)
    at /home/engineer/node_modules/homebridge-vera/VeraLink.js:208:31
    at tryCatcher (/home/engineer/node_modules/homebridge-vera/node_modules/request-promise/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/home/engineer/node_modules/homebridge-vera/node_modules/request-promise/node_modules/bluebird/js/release/promise.js:510:31)
    at Promise._settlePromise (/home/engineer/node_modules/homebridge-vera/node_modules/request-promise/node_modules/bluebird/js/release/promise.js:567:18)
    at Promise._settlePromise0 (/home/engineer/node_modules/homebridge-vera/node_modules/request-promise/node_modules/bluebird/js/release/promise.js:612:10)
    at Promise._settlePromises (/home/engineer/node_modules/homebridge-vera/node_modules/request-promise/node_modules/bluebird/js/release/promise.js:691:18)
    at Async._drainQueue (/home/engineer/node_modules/homebridge-vera/node_modules/request-promise/node_modules/bluebird/js/release/async.js:138:16)
    at Async._drainQueues (/home/engineer/node_modules/homebridge-vera/node_modules/request-promise/node_modules/bluebird/js/release/async.js:148:10)
    at Async.drainQueues (/home/engineer/node_modules/homebridge-vera/node_modules/request-promise/node_modules/bluebird/js/release/async.js:17:14)
    at process._tickCallback (node.js:415:13)`

My dimmer info in Vera is as follows:

Capabilities    211,156,0,4,17,4,L,R,B,RS,|38:1,39,43,44,114,115,119,133,134,145,
Version 3,3,52,0,7
Manufacturer    Leviton
Model   RZI06

device_type: urn:schemas-upnp-org:device:DimmableLight:1
device_file: D_DimmableLight1.xml

Any help would be much appreciated.

**Comment by damianxd** [comment]
Are you sure you're using the latest version? it should be 0.5.2

**Comment by briandye** [comment]
Hi Damian, thanks for your work -- I am on the latest version. homebridge-vera@0.5.2 running as standalone.

**Comment by damianxd** [comment]
uhm... something must be wrong because on your error it says:

> Unhandled rejection TypeError: Object dimmer has no method 'includes' at /home/engineer/node_modules/homebridge-vera/lib/functions.js:159:55 at Array.forEach 

But there is no forEach at line 159 on the latest version... how are you running this?

**Comment by briandye** [comment]
Running it standalone -- DEBUG=* node VeraLink.js

line 159 is inside of a forEach that starts on line 153. I was actually able to make it work and accept the dimmer without error but I had to comment out lines 159-163 that are looking for a "fan" device. I don't have any of these devices and none of my names contain the word fan so I am not sure what is getting hung up. Here is what I commented out:

`if (device.name.toLowerCase().includes("fan")){
                            var Fan    = require("./types/fan.js")(HAPnode,config,module);
                            HAPnode.debug('------ Fan Added: %s', device.name + ' ID:' +device.id);
                            accessories.push(Fan.newDevice(device));
                        } else `
